env.yml

```diff
@@ -1,11 +1,17 @@
-name: base
+name: mle-dev
 channels:
   - defaults
+  - conda-forge
 dependencies:
-  - python=3.10
-  - flake8
-  - isort
-  - numpy
+  - python=3.8.18
   - pandas
-  - black
+  - numpy
   - scikit-learn
+  - matplotlib
+  - scipy
+  - flake8
+  - black
+  - isort
+  - pip
+  - pip:

```

Comment:
Have you manually added these contents to `eny.ml`.?

Ideally, the best practice is to.. 
1. install the required libraries in your _local conda env_.
2. export the _env_ to `env.yml`.
3. push the new `env.yml` to the _repo_.

Reply:
Exported the environment to env.yml and push to repo